hardhat.config.js

```diff
@@ -75,13 +92,13 @@ module.exports = {
     maticBetaMainnet: {
```

Comment:
Remove matic beta mainnet and maticTestnetV3

Reply:
done